emails/2023-08-11.md

```diff
@@ -0,0 +1,25 @@
+This is the weekly CEO update from [DoltHub](https://www.dolthub.com/). I'm Tim, the CEO of DoltHub. 
+
+Last week when we were trying to come up for a name for the [React + GraphQL + Dolt stack](https://www.dolthub.com/blog/2023-08-02-workbench-architecture-and-rgd-stack/) and coined "the rugged stack", I joked that people that work with Dolt are rugged because they do a lot of yard work, branches and logs and stuff. Better than GPT?
+
+### Wordpress Works on Dolt
+
+We are out to prove to the world that Dolt "just works" with your favorite tools. In service of this mission, [Neil](https://www.dolthub.com/team#neil) published a blog on how the world's most popular website builder, [Wordpress, works with Dolt](https://www.dolthub.com/blog/2023-08-04-wordpress-on-dolt/). Neil also shows off a clever use of Dolt tags to debug what Wordpress is actually doing when he navigates the website.
+
+[![Wordpress Dolt](../images/wordpress_dolt.png)](https://www.dolthub.com/blog/2023-08-04-wordpress-on-dolt/)
+
+### Temporal Databases
+
+I'm back at it again with another search focused [blog on Temporal Databases](https://www.dolthub.com/blog/2023-08-07-temporal-database/). This article explains SQL:2011 and provides a survey of some of the SQL database that implement it. It also covers some NoSQL options.
+
+[![Temporal Database](../images/temporal-featured.png)](https://www.dolthub.com/blog/2023-08-07-temporal-database/)
+
+This article was fun to write because I had heard of temporal tables before but not temporal databases. I had to research a whole new space to write it. Hopefully you learn something from it too.
```

Comment:
```suggestion
This article was fun to write because I had heard of temporal tables before but not temporal databases. I had to research a whole new space to write it. Hopefully, you learn something from it too.
```